scribejava-apis/src/test/java/com/github/scribejava/apis/examples/OdnoklassnikiExample.java

```diff
@@ -18,14 +18,14 @@
 
     public static void main(String... args) {
         // Replace these with your client id and secret
-        final String clientId = "your client id";
-        final String publicKey = "your api secret";
-        final String clientSecret = "your client secret";
```

Comment:
Ok, agreed with this change ;-)
according to OK docs, OAuth 'client_secret' is OK 'application secret key'.
And here we should provide data in OK terms. mapping to OAuth terms will be on the next lines
```
.apiSecret(secretKey)
````